cmd/client/lib/cli.go

```diff
@@ -131,10 +137,19 @@ func Create(c *cli.Context, withInstrumentation bool, opts ...client.Option) (cl
 		}
 		opts = append(opts, client.WithChainHash(hash))
 	}
+
 	if c.Bool(InsecureFlag.Name) {
 		opts = append(opts, client.Insecurely())
 	}
 
+	if c.Bool(DecouplePrevSigFlag.Name) {
+		opts = append(opts, client.DecouplePrevSig())
+	}
+
+	if c.Bool(InsecureFlag.Name) {
+		opts = append(opts, client.DecouplePrevSig())
+	}
```

Comment:
is this expected?

Reply:
No, It is not! Thanks for the review. Fixed!